RandonSRProctor/KARPC_alpha
# 2-set-up-react-app

[Trello Link](https://trello.com/c/fmNUWRC7/2-set-up-react-app)

## Code Changes

- Created React app with Vite's builder; used the `react-ts` template
- Added Tailwind configuration files
- Added `.prettierrc` file at root of client package for so that we can have Tailwind linting enabled.
- Initialized package at root of project so that I could...
- ...add scripts the root of the package to start the server and client app without having to dig deep.
- Added very basic shell of React app to have something to show and test work.

## Test Steps

- Pull latest from repo
- Open terminal and cd down to `client/karpc-alpha-client-app` and run `npm install`
- cd back up to root of project
- Run `npm run dev/client-app`
- See that code successfully builds and runs React app.
